src/main/kotlin/org/stellar/walletsdk/utils/accountAvailableNativeBalance.kt

```diff
@@ -0,0 +1,14 @@
+package org.stellar.walletsdk.utils
+
+import java.math.BigDecimal
+import org.stellar.sdk.responses.AccountResponse
+
+fun accountAvailableNativeBalance(account: AccountResponse): String {
+  val nativeAmount = account.balances.find { it.assetType == "native" }!!.balance.toBigDecimal()
+  val reservedBalance = accountReservedBalance(account).toBigDecimal()
+  val availableAmount = nativeAmount.minus(reservedBalance)
+
+  return if (availableAmount <= BigDecimal(0)) {
+    return "0"
+  } else availableAmount.toPlainString()
```

Comment:
`availableAmount` will never be less than 0 because the amount reserved can at most be the total amount held.

Reply:
The check is there just for safety. I don't think it hurts to have it there. If you feel strongly against it, we can remove it.